customer-satisfaction/config.yaml

```diff
@@ -1,12 +1,15 @@
 enable_cache: False
+
 extra:
   tags: zenml-projects
```

Comment:
not sure we need the 'extra' tags anymore?

Reply:
will remove it